api/converter/to_bytes.go

```diff
@@ -43,53 +50,65 @@ func toJSONElement(elem json.Element) *api.JSONElement {
 	case *json.Primitive:
 		return toPrimitive(elem)
 	case *json.Text:
-		return toText(elem)
+		return toText(elem), nil
 	case *json.RichText:
-		return toRichText(elem)
+		return toRichText(elem), nil
 	case *json.Counter:
 		return toCounter(elem)
+	default:
+		return nil, fmt.Errorf("%s: %w", elem, ErrUnsupportedElement)
```

Comment:
`elem` is an instance of struct `json.Element`, will this be properly formatted?

Reply:
Oh! my mistake. The log is very messy when output like this. How about modifying it like the code below?

```go
fmt.Errorf("%v: %w", reflect.TypeOf(elem), ErrUnsupportedElement)
```

Output example:
`*json.Primitive: unsupported element`

Reply:
👍 The message got better.